roles/mesos/defaults/main.yml

```diff
@@ -11,7 +11,7 @@ mesos_follower_port: 5051
 mesos_leader_cmd: mesos-master
 mesos_follower_cmd: mesos-slave
 mesos_executor_registration_timeout: 10mins
-mesos_resources: "ports(*):[4000-5000, 31000-32000]"
+mesos_resources: "ports(*):[4000-5000, 7000-8000, 9000-10000, 25000-26000, 31000-32000]"
```

Comment:
hey, if we put `31000-32000` first in the list, would tasks prefer those ports? As is, they'll be used last, which might be the opposite of the intent here.